### PR TITLE
Add transceiver module temp thresholds

### PR DESCRIPTION
**Pull request by dplore**
### Change Scope

* Add upper and lower temperature thresholds for transceiver modules.  
* This change is backwards compatible.
### Platform Implementations

 * Arista EOS [show interfaces transceiver dom thresholds](https://www.arista.com/en/um-eos/eos-transceiver-performance-monitoring#unique_1157600725)
 * Cisco IOS XR [show controllers optics](https://www.cisco.com/c/en/us/td/docs/iosxr/cisco8000/Interfaces/73x/configuration/guide/b-interfaces-config-guide-cisco8k-r73x/m-configuring-controllers-8k.html)
 * JunOS [show interfaces diagnostics optics](https://www.juniper.net/documentation/us/en/software/junos/interfaces-ethernet-switches/topics/ref/command/show-interfaces-diagnostics-optics-ex-series.html)

### Model tree view

```
        |  +--rw oc-transceiver:thresholds
        |     +--ro oc-transceiver:threshold* [severity]
        |        +--ro oc-transceiver:severity    -> ../state/severity
        |        +--ro oc-transceiver:state
        |           +--ro oc-transceiver:severity?                   identityref
        |           +--ro oc-transceiver:laser-temperature-upper?    decimal64
        |           +--ro oc-transceiver:laser-temperature-lower?    decimal64
        |           +--ro oc-transceiver:output-power-upper?         decimal64
        |           +--ro oc-transceiver:output-power-lower?         decimal64
        |           +--ro oc-transceiver:input-power-upper?          decimal64
        |           +--ro oc-transceiver:input-power-lower?          decimal64
        |           +--ro oc-transceiver:laser-bias-current-upper?   decimal64
        |           +--ro oc-transceiver:laser-bias-current-lower?   decimal64
        |           +--ro oc-transceiver:supply-voltage-upper?       decimal64
        |           +--ro oc-transceiver:supply-voltage-lower?       decimal64
        |           +--ro oc-transceiver:module-temperature-lower?   decimal64   <-- added
        |           +--ro oc-transceiver:module-temperature-upper?   decimal64   <--added
```

### CODE REVIEW

File: release/models/platform/openconfig-platform-transceiver.yang [hunk 1]
```diff
@@ -843,6 +849,22 @@ module openconfig-platform-transceiver {
       }
       units volts;
     }
+    leaf module-temperature-lower {
```
Comment (aashaikh) [inline release/models/platform/openconfig-platform-transceiver.yang:852]: is there any significance to "module" here?  Can this just be `temperature-low` and `temperature-high`?

**Review by aashaikh — APPROVED** [review]
Pls see my question/suggestion re naming of these leaves.  LGTM otherwise.

File: release/models/platform/openconfig-platform-transceiver.yang (hunk 1 above)
Comment (dplore) [reply release/models/platform/openconfig-platform-transceiver.yang:852]: I used module to differentiate from `laser-temperature-*`

**Comment by sulrich** [comment]
@jnelson-sj @rolandphung for viz

**Pull request merged**